styles/main.css

```diff
@@ -275,17 +276,32 @@ body {
     
     nav a:hover{
         border: 0;
-        border-bottom: 1px dotted purple;
+        text-decoration: underline;
     }
     
-    header nav .burger{
-        display: float;
+    .burger{
         width: 40px;
-        height: 7px;
+        height: 25px;
         margin-left: auto;
         margin-right: auto;
         border-top: 5px solid black;
         cursor: pointer;
+        position: relative;
+        margin-top: 1em;
+    }
+    .burger::before {
+        content: "";
+        width: 40px;
+        border-top: 5px solid black;
+        position: absolute;
+        top: 5px;
+    }
+    .burger::after {
+        content: "";
+        width: 40px;
+        border-top: 5px solid black;
+        position: absolute;
+        top: 15px;
     }

```

Comment:
Here's how to get the job done with just a single div.  (Make clever use of the ::before and ::after pseudo elements to add the extra two bars needed.)